test/ProgressOnderwijsUtils.Tests/MetaObjectBulkCopyTest.cs

```diff
@@ -33,6 +30,38 @@ public sealed class BlaOk2 : ValueBase<BlaOk2>, IMetaObject, IPropertiesAreUsedI
             public string Bla2 { get; set; }
         }
 
+        public sealed class CustomBla
+        {
+            readonly string value;
+            public CustomBla(string value)
+            {
+                this.value = value;
+            }
+            public string AsString => value;
+            [MetaObjectPropertyLoader]
+            public static CustomBla MethodWithIrrelevantName(string value) => new CustomBla(value);

```

Comment:
nu zou ik verwachten dat deze method niet meer in deze class hoeft te staan; is dat zo?

Reply:
Only the method/name/ is irrelevant. There should be one method with the attribute [MetaObjectPropertyLoader].

Reply:
but why; the returnvalue already determines the type, why does it need to be in the type as well?

Reply:
Cause that's the way we find this method. Otherwise we should look for all methods on all classes (in what scope?) that have this attribute, return this objecttype and support only one simpletype parameter?

Reply:
Ah, I also forgot the parameter. The disadvantage is that we cannot initiate external classes this way, buy as discussed irl, that doesn't seem to be worth the complexity